csaf_2.1/prose/edit/src/safety-security-and-data-protection.md

```diff
@@ -31,4 +31,16 @@ Moreover, it SHALL be treated as unsafe (user) input.
   > Additional, supporting mitigation measures like retrieving only CSAF documents from trusted sources and check their integrity and
   > signature before parsing the document SHOULD be in place to reduce the risk further.
 
+The distribution requirements of CSAF data allow to specify domains as the value of the HTTP header `Access-Control-Allow-Origin`.
+While a wildcard (`*`) as header value usually prevents implementing browsers from sending credentials during the CORS request,
+the restriction to specified domains often enables sending credentials.
+As such configuration is more complex, there is a higher risk of misconfiguration.
+Furthermore, this might leak information about internal structures or tools used.
```

Comment:
Better: `Specifying a fixed single domain will let others know which domain can send javascript code that is allowed to send credentials.`

Reply:
It is not only about "who can sen credentials. Those domains (e.g. webview.example.com, secvisogram.example.com) might also leak the tools used, e.g. based on the domain name.

addressed in [a78c83c](https://github.com/oasis-tcs/csaf/pull/903/commits/a78c83c2c936e6719f925a2f84fd2eabf545cffa)

Reply:
Good enough. :)

(That a server trusts Javascript code coming from a specific domain to send credential can be a hint that this tools used by the server admnistrators as well and if an internal domain is allowed then it also reveals a bit of internal structure.)